examples/python/alphabet/alphabet.py

```diff
@@ -14,8 +14,6 @@
 
 
 import struct
-import pickle
-
```

Comment:
You should keep the space between "stdlib" imports and "local/private/custom stuff" imports.